aiogram_dialog/message_manager.py

```diff
@@ -169,6 +169,14 @@ async def send_media(self, bot: Bot, new_message: NewMessage):
             raise ValueError(
                 f"ContentType {new_message.media.type} is not supported",
             )
+        if new_message.media.type == ContentType.STICKER:
```

Comment:
let's chek if caption is passed instead. Implicitly ignoring text for one specific case is not ok